inception/inception-search-core/src/main/java/de/tudarmstadt/ukp/inception/search/SearchService.java

```diff
@@ -109,5 +109,13 @@ long determineNumOfQueryResults(User aUser, Project aProject, String aQuery,
             AnnotationFeature aAnnotationFeature)
         throws ExecutionException, IOException;
 
+    public StatisticsResult getProjectStatistics(User aUser, Project aProject, String aStatistic,
+            Double aLowerDocSize, Double aUpperDocSize)
+        throws IOException, ExecutionException;
+
+    public StatisticsResult getQueryStatistics(User aUser, Project aProject, String aStatistic,
```

Comment:
Is it possible to add some documentation what this method does? Especially the Double are difficult to understand